### DISCUSSION TITLE
编译的时候，golang好像总是下载不了，该怎么解决？

### DISCUSSION THREAD

**Discussion by sss132919** [coolsnowwolf/lede#11843] [Bugs / 问题反馈]
go: downloading github.com/spf13/cobra v1.6.1
go: downloading go.uber.org/zap v1.24.0
go: downloading google.golang.org/protobuf v1.28.1
github.com/google/go-cmp@v0.5.9: Get "https://proxy.golang.org/github.com/google/go-cmp/@v/v0.5.9.mod": dial tcp 172.217.160.113:443: connect: connection refused

Building targets
no Go files in /home/ubuntu-vm/documentself/lede/lede/build_dir/target-x86_64_musl/v2dat-47b8ee51fb528e11e1a83453b7e767a18d20d1f7/.go_work/build

**Comment by aiamadeus** [comment]
这个是网络问题，你可以用 goproxy

**Comment by djhons** [comment]
export ALL_PROXY=socks5://127.0.0.1:1080
git config --global http.proxy 'socks5://127.0.0.1:1080' 
git config --global https.proxy 'socks5://127.0.0.1:1080'
用上面的命令设置代理，用下面的命令进行编译。
GO111MODULE=on GOPROXY=https://goproxy.cn make V=s -j1
